{{cookiecutter.project_name}}/.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,118 @@
+name: Bug report
+description: Report something that is broken or incorrect
+labels: bug
+body:
+    - type: markdown
+      attributes:
+          value: |
+              **Note**: Please read [this guide](https://matthewrocklin.com/blog/work/2018/02/28/minimal-bug-reports)
+              detailing how to provide the necessary information for us to reproduce your bug. In brief:
+                * Please provide exact steps how to reproduce the bug in a clean Python environment.
+                * In case it's not clear what's causing this bug, please provide the data or the data generation procecure.
+                * Sometimes it is not possible to share the data but usually it is possible to replicate problems on publicly
+                  available datasets or to share a subset of your data.
+
+    - type: textarea
+      id: description
+      attributes:
+          label: Description of the bug
+          description: A clear and concise description of what the bug is.
+      validations:
+          required: true
+
+    - type: textarea
+      id: repex
+      attributes:
+          label: Minimal reproducible example
+          description: A code snippet that we can copy&paste to reproduce the bug without having any data
+          render: python
+          placeholder: |
+              import scanpy as sc
+              adata = sc.datasets.pbmc3k()
+
+              sc.do_something_that_doesnt_work(adata)
+
+    - type: textarea
+      id: stacktrace
+      attributes:
+          label: The error message produced by the code above
+          description: Please paste the entire error message
+          render: pytb
+          placeholder: |
+              ---------------------------------------------------------------------------
+              AssertionError                            Traceback (most recent call last)
+              <ipython-input-5-78750baef17a> in <module>
+              ----> 1 adata = sc.do_something_that_doesnt_work(adata)
+
+              /opt/conda/lib/python3.8/site-packages/scanpy/_preprocessing.py in do_something_that_doesnt_work(adata)
+                  438
+              --> 439     raise AssertionError("Demo function that fails")
+                  440
+
```

```suggestion
name: Bug report
description: Report something that is broken or incorrect
labels: bug
body:
    - type: markdown
      attributes:
          value: |
              **Note**: Please read [this guide](https://matthewrocklin.com/blog/work/2018/02/28/minimal-bug-reports)
              detailing how to provide the necessary information for us to reproduce your bug. In brief:
                * Please provide exact steps how to reproduce the bug in a clean Python environment.
                * In case it's not clear what's causing this bug, please provide the data or the data generation procecure.
                * Sometimes it is not possible to share the data but usually it is possible to replicate problems on publicly
                  available datasets or to share a subset of your data.

    - type: textarea
      id: report
      attributes:
          label: Report
          description: A clear and concise description of what the bug is.
      validations:
          required: true

```

@grst, would cutting it down to this be enough? Could include a bit more.